tests/functional/suite_test.go

```diff
@@ -167,8 +186,13 @@ var _ = BeforeSuite(func() {
 		Kclient: kclient,
 		Log:     ctrl.Log.WithName("controllers").WithName("Manila"),
 	}).SetupWithManager(k8sManager)
+
+	Expect(err).NotTo(HaveOccurred())
 	Expect(err).ToNot(HaveOccurred())
```

Comment:
you can drop one of these

Reply:
oh yeah, they're basically the same thing :D